src/ui/interfaces/FlowSelectorInterfaces.ts

```diff
@@ -1,6 +1,7 @@
 export interface Flow {
     [index: string]: PermittedNavigation | undefined,
     confirmation: PermittedNavigation,
+    verificationRequirement: PermittedNavigation,
```

Comment:
This question is a good argument for simplifying the `Flow` interface, and probably the controller generally, but for now...

Is this screen rendered irrespective of how many verification options there are?

Reply:
yes it is.